src/sounds.c

```diff
@@ -494,16 +494,13 @@ TbBool init_sound(void)
     else
       snd_settng->max_number_of_samples = 16;
     snd_settng->danger_music = 0;
-    snd_settng->no_load_music = 0;
+    snd_settng->no_load_music = 1;
```

Comment:
I presume this is midi, but do you know if it still plays the cd music?

Reply:
Both OGG and CD music still play.